src/vs/workbench/parts/debug/browser/debugStatus.ts

```diff
@@ -24,12 +25,14 @@ export class DebugStatus extends Themable implements IStatusbarItem {
 	private label: HTMLElement;
 	private icon: HTMLElement;
 	private showInStatusBar: string;
```

Comment:
showInStatusBar should be a ''never' | 'always' | 'onFirstSessionStart' | 'whenViewHidden';